stock_picking_validate_jobcan_wf/models/stock_picking.py

```diff
@@ -25,9 +25,22 @@ def _is_outgoing(self):
             "transit",
         ] and self.location_dest_id.usage not in ["internal", "transit"]
 
+    def _receipt_return_picking(self):
+        self.ensure_one()
+        if not self.move_ids:
+            return False
+        for move in self.move_ids:
+            if not (move.origin_returned_move_id and move._is_out()):
+                return False
+        return True
+
     def _compute_show_skip_jobcan_wf(self):
         for pick in self:
-            if not pick._is_outgoing() or pick.move_line_ids.mapped("owner_id"):
+            if (
+                not pick._is_outgoing()
+                or pick.move_line_ids.mapped("owner_id")
+                or pick._receipt_return_picking()
+            ):
                 pick.show_skip_jobcan_wf = False
```

Comment:
```suggestion
            if (
                not pick._is_outgoing()
                or pick._receipt_return_picking()
                or pick.move_line_ids.mapped("owner_id")
            ):
                pick.show_skip_jobcan_wf = False
                pick.skip_jobcan_wf = True
```

Reply:
Adding `pick.skip_jobcan_wf = True` and make `skip_jobcan_wf` as a compute field is not worked as expected behavior because this compute function is shared for both store and non store field. Can we just keep previous design because sharing compute function for store and non-store fields is not also a good practice from technical POV.